share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -4,91 +4,63 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
 
-
 ## Other Interesting Taps
 
-*   [osx-cross/avr](https://github.com/osx-cross/homebrew-avr)
-    - GNU AVR toolchain (Libc, compilers and other tools for Atmel MCUs, useful for Arduino hackers and AVR programmers).
-
-*   [titanous/gnuradio](https://github.com/titanous/homebrew-gnuradio)
-    -  GNU Radio and friends running on OS X.
+*   [InstantClientTap/instantclient](https://github.com/InstantClientTap/homebrew-instantclient): A tap for Oracle Instant Client. The packages need to be downloaded manually.
 
-*   [besport/ocaml](https://github.com/besport/homebrew-ocaml)
-    - A tap for Ocaml libraries, though with caveats, it requires you install its customized ocaml formula. Perhaps a template for more work.
+*   [besport/ocaml](https://github.com/besport/homebrew-ocaml): A tap for Ocaml libraries, though with caveats, it requires you install its customized ocaml formula. Perhaps a template for more work.
 
-*   [petere/postgresql](https://github.com/petere/homebrew-postgresql)
-    - Allows installing multiple PostgreSQL versions in parallel.
+*   [osx-cross/avr](https://github.com/osx-cross/homebrew-avr): GNU AVR toolchain (Libc, compilers and other tools for Atmel MCUs, useful for Arduino hackers and AVR programmers).
 
-*   [edavis/emacs](https://github.com/edavis/homebrew-emacs)
-    - A tap for Emacs packages.
+*   [petere/postgresql](https://github.com/petere/homebrew-postgresql): Allows installing multiple PostgreSQL versions in parallel.
 
-*   [InstantClientTap/instantclient](https://github.com/InstantClientTap/homebrew-instantclient)
-    - A tap for Oracle Instant Client. The packages need to be downloaded manually.
+*   [titanous/gnuradio](https://github.com/titanous/homebrew-gnuradio):  GNU Radio and friends running on OS X.
 
 ## Interesting Branches (aka forks)
 
-*   [mistydemeo/tigerbrew](https://github.com/mistydemeo/tigerbrew)
-    - Experimental Tiger PowerPC version
-
-*   [homebrew/linuxbrew](https://github.com/Homebrew/linuxbrew)
-    - Experimental Linux version
+*   [mistydemeo/tigerbrew](https://github.com/mistydemeo/tigerbrew): Experimental Tiger PowerPC version
 
-*   [wilmoore/homebrew-home](https://github.com/wilmoore/homebrew-home)
-    - Homebrew install for those that like to Homebrew @ $HOME (i.e. ~/.homebrew).
+*   [homebrew/linuxbrew](https://github.com/Homebrew/linuxbrew): Experimental Linux version
 
-*   [nddrylliog/homebrew-mingw](https://github.com/nddrylliog/homebrew-mingw)
-    - An experimental port of Homebrew for Windows (with an MSYS/MinGW environment).
+*   [wilmoore/homebrew-home](https://github.com/wilmoore/homebrew-home): Homebrew install for those that like to Homebrew @ $HOME (i.e. ~/.homebrew).
 
+*   [nddrylliog/homebrew-mingw](https://github.com/nddrylliog/homebrew-mingw): An experimental port of Homebrew for Windows (with an MSYS/MinGW environment).
```

Comment:
I'd be tempted to get rid of this one. It hasn't been updated in a *loooooong* time.